### PR TITLE
add customizer option, func to return hero page, partial for hero page

### PR DESCRIPTION
**Pull request by alex-418**
No styling, just functionality and structure 

hero

### CODE REVIEW

**Comment by alex-418** [comment]
page_select

File: resources/views/layouts/front.blade.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 @include('partials.header')
 <div class="wrap container" role="document">
 	<main class="main">
+		@include('partials.front-page-hero')
 		@yield('content')
```
Comment (bdolor) [inline resources/views/layouts/front.blade.php]: revert this change...leave this as `@yield('content')`

File: resources/views/front-page.blade.php [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 @section('content')
 	<span itemscope itemtype="http://schema.org/ItemPage">
-		<div class="row mt-sm-4">
+        <div class="row mt-sm-4">
```
Comment (bdolor) [inline resources/views/front-page.blade.php:6]: somewhere in `front-page.blade.php` add `@include('partials.front-page-hero')` (likely not in a col-md-4, for instance)

File: resources/views/partials/front-page-hero.blade.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="row">
+    <div class="col-md-6">
+        <h2>{!! $get_hero_page->post_title !!}</h2>
```
Comment (bdolor) [inline resources/views/partials/front-page-hero.blade.php]: I think we'll want to process titles through `htmlspecialchars` on titles, no? Not certain what benefit we derive leaving it un-escaped with `!!` Same consideration should be given to other content below.

**Review by bdolor — CHANGES_REQUESTED** [review]

File: resources/views/partials/front-page-hero.blade.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<div class="row">
+    <div class="col-md-6">
+        <h2>{!! htmlspecialchars($get_hero_page->post_title) !!}</h2>
```
Comment (bdolor) [inline resources/views/partials/front-page-hero.blade.php]: please refer to documentation on the correct use of `!!` https://laravel.com/docs/master/blade#displaying-data Title is 'Displaying Unescaped Data'

Comment (alex-418) [reply resources/views/partials/front-page-hero.blade.php]:
```suggestion
        <h2>{{ $get_hero_page->post_title }}</h2>
```

Comment (bdolor) [reply resources/views/partials/front-page-hero.blade.php]: cool...any related thoughts on leaving it un-escaped in the next three elements?

Comment (alex-418) [reply resources/views/partials/front-page-hero.blade.php]: Yes, looks like not a good idea... here's the results: results

Although can do it for the get_permalink... will commit now

**Review by bdolor — APPROVED** [review]
